### PR TITLE
Soc2017 ideas

### PR DESCRIPTION
**Pull request by chriscool**
In the first commit, I just copied the 2016 Ideas page and remove most of the ideas.
Feel free to add them back if you think they are interesting.
I also replaced gmane references with public-inbox. 

In the second commit I added some ideas that I had already posted on issue #188, and also one I copied from the 2014 ideas page.

### CODE REVIEW

**Comment by chriscool** [comment]
I merged this for now as it will be discussed anyway on the list when it will be time to think about the next GSoC.

File: SoC-2017-Ideas.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+* Include what is suggested in
+  [the GSoC Student Guide](http://write.flossmanuals.net/gsocstudentguide/writing-a-proposal/)
+
+([public-inbox](https://public-inbox.org/git/) can be
+used for searching the mailing list and linking to previous
+discussions.)
+
+## Note about the number of slots
+
+In 2016, the Git organization has very limited mentoring capacity.
```
Comment (gitster) [inline SoC-2017-Ideas.md]: 2017 you mean?

Comment (chriscool) [inline SoC-2017-Ideas.md]: Yeah I forgot to change this instance.

**Comment by chriscool** [comment]
Thanks, it's fixed in f3559b444197b214ea9b51f8afd6edcf35a0e90a

**Pull request merged**